docs/the-big-picture.md

```diff
@@ -0,0 +1,107 @@
+
+The architectural design is centered on streams of data through the system rather than the relationships and connections between applications. Data is sent asynchronously and applications react to events as they occur. This ability to think abstractly about streams of data as the system evolves regardless of the technical implementation is very valuable. 
+
+Say the tickets endpoint was initially written in a language and framework that can’t handle the increased volume as it scales. It can be easily replaced, with the new application simply taking over the event subscription and publication to the topic. Topics can be easily updated.
+
+A reactive architecture using an event hub like Kafka enables an increased flexibility and the ability to debug, monitor, and maintain the backend in a microservices architecture. Even so, something about this architecture feels disjointed and unaligned to reactive principles because of the real time updates.
+
+## Reactive Interaction Gateway
+The application handing those connections in our diagram will have trouble handling thousands of concurrent users with real time updates, whether they be mobile or browser based. 
+
+At scale, this poses problems in the infrastructure and architecture. Events will need to be sent via the event hub and an application has to function as an interface with the client side. Which application should do it? Should the application that handles the main business logic also handle connections?
+
+The Reactive Interaction Gateway (RIG) was designed to solve this problem elegantly and in line with Reactive principles. Using the Erlang VM, BEAM, we can model web socket connections using actors, which are much lighter weight than OS threads. 
+
+RIG functions as an application interface layer and works as an event hub for the front end.  It powers real time updates and decouples the backend interface from the frontend while enabling many concurrent users. It handles asynchronous events streaming from the event hub or from the UI.
+
+This architecture can evolve in complexity as features are built, adding or subtracting services in order to reflect the problem domain. It makes possible the continuous deployment of services without effecting users’ connections.
+
+RIG is designed to have all connections to the application be language agnostic. Routes are defined using a configuration file or by POSTing directly to the application. This gives an architect a great deal of flexibility to choose the tools they use to meet functional requirements.
+
+![simple-add-rig](assets/simple-add-rig.png)
+
+## Next Layer of Complexity: A Reference Architecture
+In the `/examples` folder, there is an example architecture made with a React frontend, RIG, a Node backend, and a Kafka instance. Here's a chart demonstrating that reference architecture in abstract. Go to [/examples](../examples) for more depth:
```

Comment:
The `/examples` link does not work

Reply:
Good catch. I've updated it to point to the Github page.